Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.py

```diff
@@ -1347,6 +1351,30 @@ def fetch_incidents():
     demisto.setLastRun({'time': current_run.isoformat().split('.')[0]})
 
 
+def add_capitalized_hash_to_context(threats_context):
+    """Add capitalized hash keys to the context such as SHA256 and MD5,
+    the keys are redundant since they are used for avoiding BC issues.
+
+    Args:
+        threats_context(list): list of dicts of context outputs for the threats of interest, each containing
+        the key 'Sha256' (and possibly (Md5)).
+
+    Returns:
+        threats_context(list): list of dicts of context outputs for the threats of interest, each containing
+        the key and value 'Sha256' (and possibly Md5) as well as the key and value 'SHA256' (and possible MD5).
+    """
+    if not isinstance(threats_context, list):
```

Comment:
Can there be a situation where the context is None or an empty dict? I would add a check for that